plugins/opsgenie/help.md

```diff
@@ -353,6 +355,7 @@ _This plugin does not contain any troubleshooting information._
 
 # Version History
 
+* 1.1.0 - Added alertId field to the output of Create Alert action
```

Comment:
```suggestion
* 1.1.0 - Added Alert ID to the Create Alert action output
```
It's important to use the user-facing 'title' attribute of the input/output when talking about it in user-facing text :)